### ISSUE TITLE
[Bug]: Player Web controls should be propper buttons

### ISSUE THREAD

**Issue by rkingett** [advplyr/audiobookshelf#2599]
### Describe the issue

    I am using the NVDA screen reader on Windows, running an instance of ABS on PikaPods, but the version is Version 2.7.2.

NVDA isn't registering the player controls as a `<button>` tag. They appear as plain text elements. Perhaps these should be changed to proper buttons so that mobile web browsers could also take advantage of the changes?

NVDA is an open source screen reader and can be found at https://github.com/nvaccess/nvda

The below explains more about clickable elements and why they might not work in all cases, especially mobile browsers.

https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML

### Steps to reproduce the issue

1. 

### Audiobookshelf version

2.7.2

### How are you running audiobookshelf?

Other

**Comment by advplyr** [comment]
Fixed in [v2.8.0](https://github.com/advplyr/audiobookshelf/releases/tag/v2.8.0)
